### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by billymohajeri**
**Accessibility report:**

- Page titles - no issues found
- Image text alternatives - Updated some alt texts [(1)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/index.html#L142) [(2)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/index.html#L146) [(3)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/index.html#L187) [(4)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/index.html#L191) [(5)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/index.html#L229) [(6)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/index.html##L233)
- Text headings - no issues found
- Color contrast - no issues found
- Resize - just 1 issue - Resolved: [(1)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/style.css#L311) [(2)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/style.css#L316) [(3)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/style.css#L549) [(4)](https://github.com/felixDev22/Mobile-first-D3/blob/040b079c7d2e27b53b7d6054a390d10a67c7153a/style.css#L563)
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @felixDev22   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
You still need to work on some issues to go to the next project, but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly change your project title to make it more professional.
    - Poor titles:

        * Microverse project | Your portfolio.
        * Portfolio project

    - Better page titles:

         * Felix's portfolio
         * Felix full-stack web developer

- [ ] Kindly make sure to have only one `h1` heading which will serve as the main heading of your website.
- [ ] The alternative text does not need to include the words "button", "link", "icon" or "image of". (Screen readers automatically provide that information.) Kindly change that. 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Portfolio Mobile First</title>
+    <title>Felix full-stack web developer</title>
```
Comment (anagudelogu) [inline index.html:8]: Nice title! 🚀

**Review by anagudelogu — APPROVED** [review]
Hi Team! @felixDev22 @billymohajeri, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Descriptive PR title and summary. 👍🏼 
✅ Descriptive commits. 💯 
✅ Nice job implementing accessibility upgrades! 🥇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — COMMENTED** [review]
# STATUS: APPROVED  
 ## Hi @felixDev22 ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**